pkg/morph/deploy/util.go

```diff
@@ -41,7 +41,9 @@ func isErrNotEnoughGAS(err error) bool {
 }
 
 func isErrInvalidTransaction(err error) bool {
-	return errors.Is(err, neorpc.ErrValidationFailed)
+	// neo-go changed -504 code to be a network fee error,
+	// but it was a verification error before v0.102.0
+	return errors.Is(err, neorpc.ErrInsufficientNetworkFee)
```

Comment:
That's a straight `isErrNotEnoughGAS`, while `isErrInvalidTransaction` is any of -5xx (you can get a lot of different codes now).

Reply:
yes, usually i split "big" libraries updates into two (or more) commits: one for the `go.mod` file and the other for the features adoptions. the problem was that `ErrValidationFailed` just does not exist anymore so it could not be complied. that is why i put it here (another -504 code, even if it is named weird for that part of the code). it is dropped right in the next commit

Reply:
Yeah, I've seen that as well.